app/views/admin/gdpr_delete_requests/index.html.erb

```diff
@@ -2,10 +2,20 @@
 <div id="gdpr-delete-requests-content" class="crayons-card overflow-admin-main-layout-padding p-4 m:p-0">
   <header class="flex flex-col l:flex-row justify-content-between l:items-center p-0 m:p-7 pb-4">
     <h1 class="crayons-title ml-3 l:ml-0">Members<span class="screen-reader-only"> (GDPR Delete Requests)</span></h1>
+    <%= form_with url: admin_gdpr_delete_requests_path, method: :get, local: true, class: "py-3" do |f| %>
+    <div class="hidden m:flex flex-col m:flex-row justify-content-between p-3 xl:px-0">
+      <div class="flex grow-1 justify-content-end">
+        <%= paginate @gdpr_delete_requests, theme: "admin", scope: @gdpr_delete_requests, label: "Paginate GDPR delete requests", context: "top" %>
+      </div>
+      <%= form_with url: admin_gdpr_delete_requests_path, method: :get, local: true, class: "grow-1 mb-3 m:mb-0" do |f| %>
+        <% render "admin/users/controls/search_field", f: f, placeholder: "Search members...", aria_label: "Search members by email or username", context: "large" %>
+      <% end %>
+    </div>
+    <div id="search-users" class="hidden mt-3">
+      <%= render "admin/users/controls/search_field", f: f, placeholder: "Search members...", aria_label: "Search members by email or username", context: "responsive" %>
+    </div>
```

Comment:
Is this for the small screen layout? I think we're currently missing the button to expand/collapse this section - will we add that in a separate PR (if so, maybe we shouldn't add this chunk of HTML yet since we don't get to see it)?

Reply:
Yes, it is! However, it doesn't appear to be working as expected--I'll pull it out of this PR and touch it up in a follow-up PR per your comment. 😊